dev-aws/kafka-shared-msk/customer-billing/topics.tf

```diff
@@ -133,3 +133,19 @@ resource "kafka_topic" "public_fulfilment_events" {
     "cleanup.policy"    = "delete"
   }
 }
+
+# this topic is used during the transition from exstream to bex bill
+# generation and contains AccountReadyToBeFulfilledEvents
+resource "kafka_topic" "transition_bex_fulfilment_request" {
+  name               = "bex.transition.fulfilment_request"
+  replication_factor = 3
+  partitions         = 5
+  config = {
```

Comment:
This also needs some `retention.bytes` or `retention.ms`, especially since it is dev

Reply:
Done